spec/user-stories.md

```diff
@@ -1,44 +1,15 @@
 ## Functional Stories
 
-### Use Case 1: Large HTTP uploads
+### Administrative Assistant
 
-As a `user` and/or a `software developer`,
-I want to upload large files that may or may/not complete due to network and/or server issues.  Further, for very large uploads also knowing what percentage of a file(s) that is uploaded is handy to have as both user and developer.
-
-#### Preconditions:
-
-User/developer is working with large files.
-
-#### Trigger:
-
-User attempts to upload a file.
-
-#### Actors:
-
- - User with a large file
- - the server to which the file needs to be uploaded to
+As a Manager I want my administrative Assistant to exercise some, but not all, of my permissions, such as updating my calendar and screening my email, so that I don't have to waste my time on these matters. Manager spends a lot of time scheduling appointments and dealing with unimportant emails. Common permission infrastructure, such a Role-Based, Attribute-Based, or Policy-Based access control, doesn't provide sufficient flexibility for Manager to offload some of this work to Assistant and Assistant to offload some of that work to others.
```

Comment:
```suggestion
As a Manager I want my administrative Assistant to exercise some, but not all, of my permissions, such as updating my calendar and screening my email, so that I don't have to waste my time on these matters. Manager spends a lot of time scheduling appointments and dealing with unimportant emails. Common permission infrastructure, such a Role-Based, Attribute-Based, or Policy-Based access control, doesn't provide sufficient flexibility for Manager to offload some of this work to Assistant and Assistant to offload some of that work to others.

See also https://github.com/w3c/lws-ucs/issues/10
```
I think it is important to show some traceability from the UC documents to the issues where they were collected (not necessarily as a 1-1 mapping though).

Reply:
We probably also want to remove the triage label on the issues that have been included in the doc (after the PR is merged).